pages/phases/assessments/written-assessments.md

```diff
@@ -0,0 +1,16 @@
+---
+layout: article
+permalink: /hiring-phases/assessments/written-assessments/
+section: hiring-phases
+sidenav: hiring-phases
+phase: assessments
+sub-phase-order: 020
+title: Written Assessments
+intro: SMEs may conduct one or two rounds of written assessments in the assessment phase.
```

Comment:
I'd rather say "During job analysis SMEs and HR may decide to turn the first structured interviews into a written assessment or demonstration project instead."

Reply:
Edited to "During job analysis, SMEs and HR may decide to do a written assessment or demonstration project instead of a first-round interview."